### PR TITLE
MS-1397 remove Pdf generation

### PR DESCRIPTION
**Pull request by jjrobothamho**
- remove Pdf generation from app

### CODE REVIEW

**Comment by zuzak** [comment]
There are a couple of mentions of PDFs in the translation strings; are these still relevant, or can they be removed?

File: apps/nrm/util/transform-data.js [hunk 1]
```diff
@@ -1,74 +0,0 @@
-'use strict';
-const moment = require('moment');
```
Comment (zuzak) [inline apps/nrm/util/transform-data.js]: `moment` doesn't seem to be used anywhere else; can we remove this as a dependency entirely?

File: apps/nrm/util/transform-data.js [hunk 2]
```diff
@@ -1,74 +0,0 @@
...
-const uuidv4 = require('uuid/v4');
-const util = require('./check-list-with-object');
```
Comment (zuzak) [inline apps/nrm/util/transform-data.js]: Can we delete `check-list-with-object` now?

File: config.js [hunk 3]
```diff
@@ -43,16 +43,10 @@ module.exports = {
   govukNotify: {
     notifyApiKey: process.env.NOTIFY_KEY || '',
     templateUserAuthId: process.env.TEMPLATE_USER_AUTHORISATION_ID || 'fe0408cb-24f0-4b5f-9bdb-6569834039fb',
-    templatePDF: process.env.TEMPLATE_PDF || 'd48b84c8-9350-4cf8-b31c-8cc4ef8237f1',
```
Comment (zuzak) [inline config.js]: Hang on, are we deleting this email entirely, or are we replacing it with an email that just doesn't have a PDF?

File: package.json [hunk 4]
```diff
@@ -11,9 +11,7 @@
     "test:local-acceptance-demo": "env BROWSER_TYPE=local BROWSER_DEMO=true _mocha --recursive acceptance-test/user-pathways",
     "lint": "eslint .",
     "build": "hof-build",
-    "watch:sass": "nodemon -e scss -x 'npm run sass'",
```
Comment (zuzak) [inline package.json]: Did these mean to get committed?

File: config.js (hunk 3 above)
Comment (jjrobothamho) [reply config.js]: I'll double check

File: apps/nrm/util/pdf-puppeteer.js [hunk 5]
```diff
@@ -1,42 +0,0 @@
-'use strict';
-/* eslint-disable no-console */
-const fs = require('fs');
-const puppeteer = require('puppeteer');
-const hogan = require('hogan.js');
```
Comment (zuzak) [inline apps/nrm/util/pdf-puppeteer.js]: Is `hogan` no longer a dependency? Can we remove it?

File: apps/nrm/util/transform-data.js (hunk 1 above)
Comment (zuzak) [reply apps/nrm/util/transform-data.js]: It's still in the package-lock.json

**Comment by jjrobothamho** [comment]
Yep the translation strings are ok; they apply to the offline form

**Pull request merged**